src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -158,6 +157,7 @@ class WorkspaceInvitePage extends React.Component {
         });
         indexOffset += personalDetailsWithoutSelected.length;
 
+        const hasUnselectedUserToInvite = this.state.userToInvite && !_.contains(selectedLogins, this.state.userToInvite.login);
```

Comment:
We can revert this too.

Reply:
It just moves the variable closer to where it is used. Should I move it back?